src/WholeBodyStateDisplay.cpp

```diff
@@ -279,6 +290,7 @@ void WholeBodyStateDisplay::loadRobotModel() {
   if (content.empty()) {
     clearRobotModel();
     setStatus(StatusProperty::Error, "URDF", "URDF is empty");
+    ROS_ERROR("URDF is empty");
```

Comment:
Why do we need to use ROS_ERROR?

Note that we use `setStatus` which should print the error message in Rviz

Reply:
It was just easier to see during debugging than opening the status when it initially segfaulted. Happy to remove

Reply:
Ok, I see, it is fine to keep it :)